<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewLoginForm</class>
 <widget class="QWidget" name="NewLoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid black;
	border-radius:10px;border:2px solid #000000;
	background-color: rgb(143, 244, 255);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>120</y>
         <width>371</width>
         <height>50</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbtYes">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>10</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbtNo">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>10</pointsize>
            <weight>10</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>-30</x>
         <y>10</y>
         <width>391</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLineEdit" name="editName">
        <property name="geometry">
         <rect>
          <x>111</x>
          <y>18</y>
          <width>231</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{background-color: rgb(255, 255, 255);	border-radius:10px;border:2px solid #000000;}
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>18</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>账户： </string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rbtStduent">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>88</y>
          <width>57</width>
          <height>19</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 10pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>学生</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rbtAdmin">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>88</y>
          <width>112</width>
          <height>19</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 10pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>管理员     </string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rbtTeacher">
        <property name="geometry">
         <rect>
          <x>177</x>
          <y>88</y>
          <width>89</width>
          <height>19</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 10pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>老师    </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>53</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editPassword">
        <property name="geometry">
         <rect>
          <x>111</x>
          <y>53</y>
          <width>231</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{background-color: rgb(255, 255, 255);	border-radius:10px;border:2px solid #000000;}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
